### PR TITLE
feat(COS-2766): Add Transfer service for requestBankList

### PR DESCRIPTION
**Pull request by LuisCuascota**

### CODE REVIEW

File: src/module/Transfer.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import { TransferService } from "service/TransferService.ts";
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { IKushki, init } from "Kushki";
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```
Comment (md02591) [inline src/module/Transfer.ts:4]: si son necesarios los eslint disable?

File: showcase/src/containers/Transfer/GetBankList/GetBankList.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
+import { ContainerDemo } from "../../../components/ContainerDemo/ContainerDemo.tsx";
+import InputConfigurationDemo from "../../../components/ConfigurationDemo/Components/InputConfigurationDemo.tsx";
+import { useState } from "react";
+import CardNumberHelper from "../../../components/CardNumberHelper/CardNumberHelper.tsx";
+import { init } from "Kushki";
+import { requestBankList } from "Kushki/Transfer";
+import { ResponseBox } from "../../../components/ResponseBox/ResponseBox.tsx";
+import "./GetBankList.css";
```
Comment (md02591) [inline showcase/src/containers/Transfer/GetBankList/GetBankList.tsx:8]: espacio

File: README.md [hunk 3]
```diff
@@ -367,3 +367,30 @@ try {
   console.error("Catch error on request card Token", error.code, error.message);
 }
 ```
+# Transfer Transactions
+## Request Bank List
+To get Bank List for Transfer transactions, you should call [`requestBankList`](https://ksh-js-sdk-dev.github.io/kushki-js-sdk/functions/Transfer.requestBankList.html) method with Kushki instance that was previously initialized with init [`init`](#library-setup) method
+
+This method is useful when the processor require bank list and if you want the customer  pay at a specific bank of your choice, more details [Click here](https://ksh-js-sdk-dev.github.io/kushki-js-sdk/functions/Transfer.requestBankList.html)
```
Comment (md02591) [inline README.md:374]: sugerencia


This method is useful in situations where the processor requires a list of banks and allows the customer to choose a specific bank to make their payment.

File: src/module/Transfer.ts [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+import { BankListResponse } from "types/bank_list_response";
+
+/**
+ * Function to get list of available banks for Transfer transactions
+ *
+ * @group Methods
+ * @param kushkiInstance - Object that that was previously initialized with {@link init} Kushki method
+ * @returns {Promise<BankListResponse>} - List of available banks
+ * @throws
+ *  - if `options.publicCredentialId` into `kushkiInstance` is not valid or the request fails then throw {@link ERRORS | ERRORS.E014}
```
Comment (md02591) [inline src/module/Transfer.ts:15]: espacio: Enter

File: src/module/Transfer.ts (hunk 1 above)
Comment (LuisCuascota) [reply src/module/Transfer.ts:4]: Es por la documentacion, para poder agregar los links a esos elementos @md02591

**Pull request merged**